mino/minogrpc/example_test.go

```diff
@@ -57,48 +58,88 @@ func ExampleRPC_Call() {
 }
 
 func ExampleRPC_Stream() {
```

Comment:
I would maintain the previous example for the sake of clarity, as it's simpler and much needed to explain the basic Stream setup.

I understand the new code is very nice to test the system, but I would possibly create a separate Example, possibly mentioning its purpose, e.g. `ExampleRPC_OpenTracingDemo` and adding some comments as to how to see all its opentracing goodness